Maple2Storage/Types/Metadata/AdditionalEffectMetadata.cs

```diff
@@ -64,9 +64,15 @@
     [XmlElement(Order = 24)]
     public EffectRideMetadata Ride;
```

Comment:
This isnt needed

Reply:
I just implemented something that uses this in another branch a few days ago because ride wasn't automatically used. I needed to manually send the mount packets for things like the young frog madrakan buff to work

Reply:
hmm this comment was supposed to be for line 71 (string Feature). not sure how it got here.

Reply:
Oh woops lol. Yeah I'm not sure how that one slipped through and got in. I think I was testing something but forgot to remove